docs/ballerina-by-example/examples/file-api/file-api.bal

```diff
@@ -1,58 +1,73 @@
 import ballerina.file;
 
 function main (string[] args) {
-    //Create 'File' struct and open for writing.
-    file:File target = {path:"/tmp/result.txt"};
-    target.open(file:W);
-
-    //Close the file once done.
-    target.close();
+    //Create 'File' struct.
+    string path = "/tmp/result.txt";
+    file:File target = {path:path};
```

Comment:
Can we put the path in the file struct initializer? Why do you need a variable here?